rest/replicator_test.go

```diff
@@ -4566,6 +4566,63 @@ func TestLocalWinsConflictResolution(t *testing.T) {
 	}
 }
 
+func TestSendToOldClientIgnoreConflictsFalse(t *testing.T) {
```

Comment:
The test name isn't hugely descriptive about what this is testing.

```suggestion
func TestSendChangesToNoConflictPreHydrogenTarget(t *testing.T) {
```